manifest.yaml

```diff
@@ -78,6 +78,10 @@ arguments:
     schema:
       type: boolean
     description: If this is set to true the `make release` step and assets declaration in .releaserc.yaml will render regardless of other arguments.
+  temporal.version:
```

Comment:
This is also defined in `stencil-outreach`. It should ideally only be defined in one place and then imported in another. 

Since now it's used here, could you also open a companion PR to import this definition into `stencil-outreach`? https://github.com/getoutreach/stencil-outreach/blob/main/manifest.yaml#L128

You can see an example of an import here:

https://github.com/getoutreach/stencil-outreach/blob/main/manifest.yaml#L9-L10

Reply:
Oh so I can import it here instead of "defining" it

Reply:
I think this one is what should be the actual source of truth tho. https://github.com/getoutreach/stencil-temporal/blob/main/manifest.yaml#L16 with that in mind, what the best play?

Reply:
Talked with Jorj about this, sounds like the best move might be to use this "module hook" stuff to render this from the `stencil-temporal` module? https://github.com/getoutreach/stencil-outreach/blob/main/templates/internal/lifecycle/_helpers.tpl#L77-L96

LMK if that sounds like the play or if you have another idea in mind

Reply:
removed the arg, and decided to link out to the generated temporal docs from stencil-temporal